components/dash-html-components/setup.py

```diff
@@ -2,22 +2,22 @@
 import json
 from setuptools import setup
 
-with open('package.json') as f:
+with open("package.json") as f:
```

Comment:
I think we can delete `components/*/setup.py` now, right? Also `MANIFEST.in`, maybe some others too... same in `dash-renderer`.

Anyway I won't make you do this now, but I'm reminded of it after seeing this file being edited.